__tests__/app.test.js

```diff
@@ -44,25 +45,30 @@ describe("GET /api/topics", () => {
         expect(response.body.msg).toBe('invalid endpoint')
       });
   });
+  test("test all topics in the test file are returned as part of the response by testing  if both of the array length equals ", () => {
+    const numOfArrayExpected = topicData.length
+    console.log(numOfArrayExpected)
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topic.length).toEqual(numOfArrayExpected
+        );
+      });
+  });
+  
 });
 
-// describe("GET /api/", () => {
-//   test("should return 200", () => {
-//     return request(app).get("/api/").expect(200);
-//   });
-//   test("should return an array of topic objects,each of which including slug and description ", () => {
-//     return request(app)
-//       .get("/api/topics")
-//       .expect(200)
-//       .then((response) => {
-//         expect(response.body.topic).toEqual(
-//           expect.arrayContaining([
-//             expect.objectContaining({
-//               slug: expect.any(String),
-//               description: expect.any(String),
-//             }),
-//           ])
-//         );
-//       });
-//   });
-// });
+describe("GET /api/", () => {
+  test("should return 200", () => {
+    return request(app).get("/api").expect(200);
+  });
+  test("should return an object, referenced from endpoints.json", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.apiInfo).toMatchObject(apiData)
```

Comment:
Please use .toEqual here to make sure the response contains exactly the same content as the file. Using .toMatchObject is great and enables flexibility, but in this case it's counterproductive.